tests/v2/validation/projects/projects.go

```diff
@@ -0,0 +1,184 @@
+package projects
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"regexp"
+	"strings"
+	"time"
+
+	v3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	"github.com/rancher/rancher/tests/framework/extensions/configmaps"
+	"github.com/rancher/rancher/tests/framework/extensions/defaults"
+	projectsApi "github.com/rancher/rancher/tests/framework/extensions/kubeapi/projects"
+	"github.com/rancher/rancher/tests/framework/extensions/kubeconfig"
+	namegen "github.com/rancher/rancher/tests/framework/pkg/namegenerator"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kwait "k8s.io/apimachinery/pkg/util/wait"
+)
+
+const (
+	roleOwner               = "cluster-owner"
+	roleProjectOwner        = "project-owner"
+	admin                   = "admin"
+	standardUser            = "user"
+	defaultNamespace        = "fleet-default"
+	RancherNamespace        = "cattle-system"
+	kubeSystemNamespace     = "kube-system"
+	localCluster            = "local"
```

Comment:
since we are using most of these constants in multiple validations, should we consider adding an extension for these? cc @igomez06 what do you think?

Reply:
I like the idea of having these constants in a extension. Will wait for Israel's input on this before making changes.